keep/api/observability.py

```diff
@@ -26,33 +27,49 @@
 def setup(app: FastAPI):
     logger = logging.getLogger(__name__)
     # Configure the OpenTelemetry SDK
-    service_name = os.environ.get("SERVICE_NAME", "keep-api")
-    otlp_collector_endpoint = os.environ.get("OTLP_ENDPOINT", False)
+    service_name = os.environ.get("OTEL_SERVICE_NAME", "keep-api")
+    otlp_collector_endpoint = os.environ.get("OTEL_EXPORTER_OTLP_ENDPOINT", False)
+    otlp_traces_endpoint = os.environ.get("OTEL_EXPORTER_OTLP_TRACES_ENDPOINT", None)
+    otlp_logs_endpoint = os.environ.get("OTEL_EXPORTER_OTLP_LOGS_ENDPOINT", None)
+    otlp_metrics_endpoint = os.environ.get("OTEL_EXPORTER_OTLP_METRICS_ENDPOINT", None)
```

Comment:
@vr-varad for the sake of backwards compatibility (assuming that some already use that), can we have default values first taken from the old env variables? see example in the next comment